### ISSUE TITLE
Port last examples from main repo

### ISSUE THREAD

**Issue by fpliger** [pyscript/examples#7]
This issue is capture and track the last reminaing examples that were not ported[ from the main repo yet](https://github.com/pyscript/pyscript/tree/main/examples):

* [ ] handtrack
* [ ] mario
* [ ] toga
* [ ] d3
* [ ] message passing
* [ ] micrograd
* [ ] repl
* [ ] repl2
* [ ] simple_bioinformatics_tool
* [ ] todo-pylist
* [ ] todo

NOTE: The resolution for each one of these can either be that we port them or that we drop the example.

**Comment by madhur-tandon** [comment]
Some of them can be ported such as `simple_bioinformatics_tool`.

For others, we might need to wait for `py-editor` to get merged...
and for some, we don't have a story around plugins...

**Comment by fpliger** [comment]
Right. I'd say we skip for now until we can actually reintroduce Python plugins.

**Comment by madhur-tandon** [comment]
I have ported two more that are not present in the list above -- 
- [x] py-jokes
- [x] panel and hvPlot

These two were being used in the homepage of pyscript.com (when someone is NOT logged in...)
Have created an associated PR here: https://github.com/anaconda/pyscript-dot-com/pull/2362 to fix 404 issues...

One more is not present in the list, again being used in the homepage of pyscript.com (when someone is NOT logged in...):
- [ ] Tic Tac Toe 

The basic blocker is being able to call a function with args via `py-click` such as:
`py-click="foo(5, 2)"`

Apparently this used to be possible with the "classic" PyScript unless I am missing something...
